account_tax_change/wizards/account_move_apply_tax_change.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+
+
+class AccountMoveApplyTaxChange(models.TransientModel):
+    _name = "account.move.apply.tax.change"
+    _description = "Apply a tax change on invoices."
+
+    @api.model
+    def default_get(self, fields_list):
+        res = super(AccountMoveApplyTaxChange, self).default_get(fields_list)
+        active_model = self.env.context.get("active_model")
+        if active_model == "account.invoice":
+            invoice_ids = self.env.context.get("active_ids")
+            invoices = self.env[active_model].search(
+                [
+                    ("id", "in", invoice_ids),
+                    ("state", "in", ("draft", "open")),
+                ]
+            )
+            res["invoice_ids"] = [(6, 0, invoices.ids)]
```

Comment:
IMO we should raise an error if one of the selected invoices is not in an "allowed" state.